examples/canonicalAllele-ex2.yaml

```diff
@@ -0,0 +1,134 @@
+id: clingen:CA415424538
+type: CategoricalVariant
+name: NC_000001.11:g.1699974C>G
+description: ...
+aliases:
+  - NC_000001.11:g.1699974C>G
+  - NC_000001.10:g.1631413C>G
+extensions:
+  - name: cytogenetic location
+    value: 1p36.33
+  - name: hgvs list
+    value:
+      - nucleotideExpression:
+          syntax: hgvs.g
+          value: NC_000001.11:g.1699974C>G
+      - nucleotideExpression:
+          syntax: hgvs.g
+          value: NC_000001.10:g.1631413C>G
+constraints:
+  - type: DefiningAlleleConstraint
+    allele:
+      id: ga4gh:VA.WjRY1EXgci6Nd0M2yI86Ue_OOnfnb-Ss
+      type: Allele
+      name: NC_000001.11:g.1699974C>G
+      description: VRS Allele of NC_000001.11:g.1699974C>G, generated with the VICC Variation Normalizer (https://github.com/cancervariants/variation-normalization).
+      digest: WjRY1EXgci6Nd0M2yI86Ue_OOnfnb-Ss
+      expressions:
+        - syntax: hgvs.g
+          value: NC_000001.11:g.1699974C>G
+      location:
+        id: ga4gh:SL.YmbtK6FLqlk2bLjpFwH-y7k4RLq_t7L3
+        type: SequenceLocation
+        digest: YmbtK6FLqlk2bLjpFwH-y7k4RLq_t7L3
+        sequenceReference:
+          type: SequenceReference
+          refgetAccession: SQ.Ya6Rs7DHhDeg7YaOSg1EoNi3U_nQ9SvO
+        start: 1699973
+        end: 1699974
+        sequence: C
+      state:
+        type: LiteralSequenceExpression
+        sequence: G
+    relations:
+      - primaryCoding:
+          code: liftover_to
+          system: ga4gh-gks-term:allele-relation
+      - primaryCoding:
+          code: transcribed_to
+          system: http://www.sequenceontology.org
+          iris:
+            - http://www.sequenceontology.org/browser/current_release/term/transcribed_to
+mappings:
+  - coding:
+      system: https://reg.clinicalgenome.org/
+      code: CA415424538
+      iris:
+        - https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=CA415424538
+    relation: exactMatch
```

Comment:
I think this may be a `closeMatch` since the deinfing allele above is not a canonical allele or categorical variation but a contextual variation.  CAIDs are canonical alleles.  I don't think it matters for the testing case, but as an example for others, it could cause confusion.